javascript/javascript3/week2/fetchAndWait.js

```diff
@@ -0,0 +1,26 @@
+console.log("Script Loaded");
+// Using Promise and .then
+
+setTimeout(() => {
+  fetch("https://yesno.wtf/api")
+    .then((response) => response.json())
+    .then((data) => {
+      console.log("Success", data);
+    }),
+    3000;
+});
+
+// using async await
+async function fetchWait() {
+  try {
+    const response = await fetch("https://yesno.wtf/api");
+    const stringify = await response.json();
```

Comment:
the name stringify is not correct. .json() converts it into a javascript object
Also, stringify is a verb